build/yamls/antrea-windows.yml

```diff
@@ -0,0 +1,197 @@
+apiVersion: v1
+data:
+  run-antrea-agent.ps1: |-
+    $ErrorActionPreference = "Stop"
+    # wins will rename the binary when executing it. So we need to copy the binary everytime before running it.
+    mkdir -force /host/k/antrea/bin
+    cp /k/antrea/bin/* /host/k/antrea/bin/
+    C:/k/antrea/utils/wins.exe cli process run --path /k/antrea/bin/antrea-agent.exe --args "--config=/k/antrea/etc/antrea-agent.conf --logtostderr=false --log_dir=/k/antrea/logs/ --alsologtostderr" --envs "KUBERNETES_SERVICE_HOST=$env:KUBERNETES_SERVICE_HOST KUBERNETES_SERVICE_PORT=$env:KUBERNETES_SERVICE_PORT ANTREA_SERVICE_HOST=$env:ANTREA_SERVICE_HOST ANTREA_SERVICE_PORT=$env:ANTREA_SERVICE_PORT NODE_NAME=$env:NODE_NAME"
+kind: ConfigMap
```

Comment:
Are able to use kustomize to share some pieces with Linux yamls, like ConfigMap and CNI conf? Or it is hard given many differences from Linux?

Reply:
ConfigMap  and CNI conf is different in windows, so can cannot share these pieces.

Reply:
I think we might be able to add patches to patch yamls for Windows? Anyway we can make the change to enable sharing later.